core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -38,6 +40,11 @@
    * Builds an instance using the driver's default implementation (based on Typesafe config), except
    * that application-specific options are loaded from a classpath resource with a custom name.
    *
+   * <p>The class loader used to locate application-specific classpath resources is {@linkplain
```

Comment:
I think we can be even more specific, to avoid any ambiguity:
```suggestion
   * <p>The class loader used to locate {@code <resourceBaseName>.*} is {@linkplain
```
And then maybe move this paragraph after the list.